printf.c

```diff
@@ -305,9 +306,9 @@ static size_t _ntoa_long(out_fct_type out, char* buffer, size_t idx, size_t maxl
 }
 
 
-/* internal itoa for 'long long' type */
+/* internal itoa for 'int64_t' type */
 #if defined(PRINTF_SUPPORT_LONG_LONG)
-static size_t _ntoa_long_long(out_fct_type out, char* buffer, size_t idx, size_t maxlen, unsigned long long value, bool negative, unsigned long long base, unsigned int prec, unsigned int width, unsigned int flags)

```

Comment:
If I'm not mistaken, `int64_t` is an optional typedef, and it's only mandated to exist since C99. C99 is not required to have 64-bit types at all, IIANM.

The thing is, @mpaland has "catered" to this issue via the `PRINTF_SUPPORT_LONG_LONG`, which, when turned off, makes this piece of code go away. Is that not satisfactory?